docs/tutorial/node/src/getting_started.md

```diff
@@ -0,0 +1,44 @@
+# Getting Started
+
+In this tutorial, we use JavaScript as the host programming language. We also support other programming languages like
+[Rust](https://slint-ui.com/docs/rust/slint/) or [C++](https://slint-ui.com/docs/cpp/).
```

Comment:
I don't think it's useful to mention other languages here.  
I can imaging that JS programmer don't like to be reminded all the time that :-)

Reply:
Should we remove that hint then also from the other tutorials?

Reply:
@ogoffart ?

Reply:
I'd say so.  But that's also fine to keep it.